README.md

```diff
@@ -1,18 +1,44 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+## My Portfolio Project
 
-> **Milestone #1 Portfolio Project!**
+> **Milestone #2 Portfolio Project!**
 
-# Project Requirements
+## Project Requirements
 
-> In this project, you will be required to build a portfolio with 2 sections only.
+**General Requirements**
 
-- the header
-- the headline section(right after the header)
+**In this project, I will be required to achieve the following.**
+
+- Understand how to parse a Figma design to create a UI.
+- Use Flexbox to place elements in the page.
+- Use CSS Grid to place elements in the page.
+- Build a personal portfolio site.
 
 > The web page similar to the one in [this link](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)
 
+## Detailed Requirements
+
+I will build the middle section of the mobile website. The sections of the mobile website should include:
+
+- the **"Works"** section
+
+- the **"About myself"** section (placed after the projects and before the contact form):
+
+**The other requirements are as follows;**
+
+- In order to lay out the elements in "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.
+
+- Please use ul and li for categories in "Work" section. Html, Css, and Javascript should be list items.
+
+- Please use ul and li for categories in "About myself" section. Languages, Frameworks, and Skills should be list items.
+
+- You don't need to implement the detail modal window that requires JavaScript.
+
+- You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using the templates in Figma.
+
+- Implement the button interactions (enable, hover, etc.).
+
```

Comment:
[OPTIONAL] - Kindly consider removing this content. At the moment, this is not required to maintain the intended purpose of the `ReadMe` file. Otherwise, great work so far. However, for future references 🤓  and to avoid having your projects invalidated, you're required to follow the [Microverse ReadMe](https://github.com/microverseinc/readme-template/blob/master/README.md) template.

Reply:
@blessedjasonmwanza Thank you for your feedback i will make the required changes. The previous reviewer asked me to remember to add a screenshot of my work done so far. could you kindly explain were exactly i should place the screenshot. Is it in the readme file or ??

Reply:
Hi @teke85 ,

Thanks for your feedback and great communication skills. You can place the screenshot of your project just below the description (on top of the `prerequisites` section).

Reply:
noted thank you.